app/src/components/containers/AttributeListFormField/attributeListFormField.jsx

```diff
@@ -54,10 +54,15 @@ export const AttributeListFormField = ({
 }) => {
   const { formatMessage } = useIntl();
 
-  const attributesCaption =
-    shown || (!shown && !value?.length)
-      ? attributesNote || formatMessage(messages.attributesNote)
-      : formatMessage(messages.attributesNotActive);
+  const getAttributesCaption = () => {
+    if (!shown) {
+      return value?.length
+        ? formatMessage(messages.attributesNotActive)
+        : attributesNote || formatMessage(messages.attributesNote);
```

Comment:
Looks like we'll provide custom `attributesNote` for all cases except the notifications tab. And if it is not provided, we'll face the bug that notifications related message shown for not only notifications related use cases.
So please lets move this specific message to the parent component and provide it via `attributesNote` prop.